cloudbuild.yaml

```diff
@@ -13,7 +13,7 @@ steps:
     id: Maven Package
     env:
       - TZ=Europe/London
-    args: ['-Dmaven.repo.local=/workspace/.m2/repository', 'package']
+    args: ['-Dmaven.repo.local=/workspace/.m2/repository', 'verify', 'package']
```

Comment:
I think it's worth chucking the how-to-run-tests type comment that you made in sdc-int-team into a CC readme. It's useful information which will just get lost if it's not more visible.

Reply:
done